depends/packages/bdb.mk

```
@@ -32,5 +32,12 @@ define $(package)_build_cmds
 endef
 
 define $(package)_stage_cmds
-  $(MAKE) DESTDIR=$($(package)_staging_dir) install_lib install_include
+  $(MAKE) DESTDIR=$($(package)_staging_dir) install
+endef
+
+define $(package)_postprocess_cmds
+  cd $($(package)_staging_dir) && \
+  cd ../../../../../../zcutil && \
```

I think this should work:
```suggestion
  cd $(BASEDIR)/../zcutil && \
```